### ISSUE TITLE
Suggestion: Add a global event notification for the Start of the Tedjin civil war

### ISSUE THREAD

**Issue by JukeJukestar** [MattTheLegoman/RealmsInExile#443]
A lot of the players Ive talked to on the discord didn't even know the Tedjin had their intricate system of events and thus never played them. Id like for them to have more attention drawn to them through a global event in addition to their bookmark.

The idea would be that once the civil war starts, either all rulers on the map, or all rulers in Harad + Sauron get an event similar to the Return of the King.
It could look somewhat like this:

"Tedjin civil war"
"The tensions within the Tedjin confederacy have finally reached their climax and led to full on civil war between the faction of the reformer Bôr and the conservative faction. How this will affect the region remains to be seen"

This could optionally be preceded by a "Tensions brewing in the Tedjin Confederacy" event only for actors who can support a faction, which would inform them they now have a decision available to intervene, so the decision doesnt just show up out of the blue.

**Comment by JukeJukestar** [comment]
Works fine as far as I can tell. can we close this? @jj248 or are you still working on it?

**Comment by JukeJukestar** [comment]
**Other Global Event Notifications we could add (If they havent been added yet)**
- Gondor Destroyed by Mordor
- Barad-Dur cast down
- Khatajate reformed/ Good Tedjin Khatajate founded
- Felaya rejoins Bellakar through negotiation
- Formation of Anadune as "Anti Gondor"
- Castamiri Reclamation
- Numenor-In-Exile founded
- Abrakhan rising (Forms its empire)
- Eotheod restored
- Greenwood restored/ Dol Guldur cleansed
- Stormhost collapse? (Once the title is destroyed)
- New Dark Lord Saruman

These are frankly a ton of options, but it would be cool if we had one for every one of the large events on Middle-Earth (Although a lot of these are quite unlikely for AI)

**Comment by RedArkady** [comment]
Putting a word in for formation of the Aladegana, as that would have significant regional consequences.

**Comment by RedArkady** [comment]
Localised everything Jamie has done so far. 'Greater Dunland' global event currently has the 'vassalised by Saruman' decision linked to it instead of the more newsworthy 'uber-Dunland' one, so that needs tweaking.

**Comment by JukeJukestar** [comment]
Yep, I dont think Dunland under saruman needs a news event - only Utrí Dunhainn

**Comment by tharondir** [comment]
````````````````````

**Comment by tharondir** [comment]
````````````````````
